src/PixelFlags.cc

```diff
@@ -219,15 +229,6 @@ void PixelFlagsAlgorithm::measure(afw::table::SourceRecord& measRecord,
     }
     fullSpans->clippedTo(mimage.getBBox())->applyFunctor(func, *(mimage.getMask()));
 
-    // Set the EDGE flag if the bitmask has NO_DATA set
-    try {
-        if (func.getAnyBits() & MaskedImageF::Mask::getPlaneBitMask("NO_DATA")) {
-            measRecord.set(_anyKeys.at("EDGE"), true);
-        }
-    } catch (pex::exceptions::InvalidParameterError& err) {
-        throw LSST_EXCEPT(FatalAlgorithmError, err.what());
-    }
-
```

Comment:
Is this related to `_anyKeys["NO_DATA"] = _anyKeys.at("EDGE");`? If so, it should probably have been a separate commit (i.e., "clean up NO_DATA handling" as opposed to "Add edgeCenter*").

Reply:
Fair point: I've made that a separate commit.